### ISSUE TITLE
Segmentar o sistema de acordo com o perfil para atender tanto a Graduação quanto a Pós-Graduação

### ISSUE THREAD

**Issue by alecostaweb** [uspdev/seta#18]

**Comment by fabiojm** [comment]
O que voce acha de tentarmos modelar o sistema para atender os 2 simultaneamente?

**Comment by alecostaweb** [comment]
Sim, esse é o objetivo. Entendo que acordamos nesse rumo? Se sim, vou dar meus pitacos na modelagem de dados que você colocou lá no drive USPdev.

**Comment by fabiojm** [comment]
Opa, isso mesmo !!

**Comment by alecostaweb** [comment]
Antes de sair modelando, pensei em algo como um Mind Map para organizarmos e sabermos quais as tabelas/entidades atacar no model. Devem ter recebido um e-mail do site https://bubbl.us/9032995

**Comment by alecostaweb** [comment]
![uspdev_seta](https://user-images.githubusercontent.com/10613364/39627354-0028361e-4f7c-11e8-894d-c53e28947e7a.png)

**Comment by fabiojm** [comment]
Fiz uma alteração no mind, poderia verificar?

**Comment by alecostaweb** [comment]
![setamindmapfabio](https://user-images.githubusercontent.com/10613364/39708150-c5434bb0-51ec-11e8-94ce-b7027e54d914.png)
4 Perfis: Administrador, Secretaria, Discente e Docente
O que está em verde indica as ações de cada perfil.
Em amarelo indica a tabela.
Só fiquei na dúvida quanto ao Docente, ele Gerencia as bancas e trabalhos acadêmicos?

**Comment by fabiojm** [comment]
Fiz uma alteração no mind, poderia verificar?

**Comment by fabiojm** [comment]
Foi separado o a graduação e pós-graduação
